applications/CompressiblePotentialFlowApplication/python_scripts/compute_forces_on_nodes_process.py

```diff
@@ -0,0 +1,54 @@
+import KratosMultiphysics
+
+def Factory(settings, Model):
+    if(not isinstance(settings, KratosMultiphysics.Parameters)):
+        raise Exception(
+            "expected input shall be a Parameters object, encapsulating a json string")
+    return ComputeForcesOnNodesProcess(Model, settings["Parameters"])
+
+# all the processes python processes should be derived from "python_process"
+
+class ComputeForcesOnNodesProcess(KratosMultiphysics.Process):
+    def __init__(self, Model, settings):
+        KratosMultiphysics.Process.__init__(self)
+
+        default_parameters = KratosMultiphysics.Parameters(r'''{
+            "model_part_name": "please specify the model part that contains the surface nodes",
+            "create_output_file": false
+        }''')
+
+        settings.ValidateAndAssignDefaults(default_parameters)
+
+        self.body_model_part = Model[settings["model_part_name"].GetString()]
+        self.create_output_file = settings["create_output_file"].GetBool()
+
+    def ExecuteFinalizeSolutionStep(self):
+        self.Execute()
+
+    def Execute(self):
+        KratosMultiphysics.Logger.PrintInfo('COMPUTE FORCES AT NODES')
+
+        KratosMultiphysics.VariableUtils().SetToZero_VectorVar(KratosMultiphysics.FORCE, self.body_model_part.Nodes)
+
+        for cond in self.body_model_part.Conditions:
+            n = cond.GetValue(KratosMultiphysics.NORMAL)
+            cp = cond.GetValue(KratosMultiphysics.PRESSURE)
+
+            for node in cond.GetNodes():
+                added_force = KratosMultiphysics.Vector(3)
+                force = KratosMultiphysics.Vector(3)
+
+                added_force = n*(cp/2.0)
+                force = node.GetValue(KratosMultiphysics.FORCE) + added_force
```

Comment:
@BasselSaridar could you change the variable FORCE to REACTION? In this manner we need to change even less things to adapt to the FSI workflow